src/components/CompareResults/TableHeader.tsx

```diff
@@ -336,39 +364,23 @@ function TableHeader({
               onToggleFilter(header.key, checkedValues)
             }
           />
-        </ButtonGroup>
-      );
-    }
-
-    if ('sortFunction' in header) {
-      return (
-        <SortableColumnHeader
-          name={header.name}
-          displayLabel={true}
-          sortDirection={header.key === sortColumn ? sortDirection : null}
-          onToggle={(newSortDirection) =>
-            onToggleSort(header.key, newSortDirection)
-          }
-        />
-      );
-    }
+        );
+      }
 
-    if ('filter' in header) {
-      return (
-        <FilterableColumnHeader
-          possibleValues={header.possibleValues}
-          name={header.name}
-          columnId={header.key}
-          uncheckedValues={filters.get(header.key)}
-          onClearFilter={() => onClearFilter(header.key)}
-          onToggleFilter={(checkedValues) =>
-            onToggleFilter(header.key, checkedValues)
-          }
-        />
-      );
-    }
-
-    return header.name;
+      return header.name;
+    })();
+    return header.tooltip ? (
+      <Tooltip
+        title={header.tooltip}
+        placement='top'
+        arrow
+        classes={{ tooltip: styles.typography }}
+      >
+        <Box sx={{ cursor: 'pointer' }}>{content}</Box>
```

Comment:
Why is the Box needed? Can you please add a comment explaining why?

Reply:
Box is used because tooltip expects a single valid element but sometimes content is a string (header.name) I can use a span here too.

Reply:
Ah I see, sounds good. Can you add a comment to the code about that ?